MobileCenter/MobileCenter/Internals/Utils/MSUtility+Date.h

```diff
@@ -0,0 +1,22 @@
+#import <Foundation/Foundation.h>
+#import "MSUtility.h"
+
+/**
+ * Utility class that is used throughout the SDK.
+ * Date part.
+ */
+@interface MSUtility (Date)
+
+/**
+ * Return the current date (aka NOW) in ms.
+ *
+ * @discussion
```

Comment:
Here, too. The order would be `@param`, `@return`, `@warning` and then `@discussion`